messages/Cargo.toml

```diff
@@ -33,3 +33,4 @@ strum = "0.16.0"
 strum_macros = "0.16.0"
 derive_builder = "0.10.2"
 tokio = { version = "1.15.0" }
+actix = "0.13.0"
```

Comment:
Was wondering if we could drop this from `messages` and turns out yes
https://github.com/hyperledger/aries-vcx/pull/680